docs/docs/10.1-animation.md

```diff
@@ -84,9 +84,42 @@ You'll notice that when you try to remove an item `ReactCSSTransitionGroup` keep
 }
 ```
 
+### Animate Initial Mounting
+
+`ReactCSSTransitionGroup` provides the optional prop `transitionAppear`, to add an extra transition phase at the initial mount of the component. There is generally no transition phase at the initial mount as the default value of `transitionAppear` is `false`. Following an example which passes the prop `transitionAppear` with the value `true`.
+
+```javascript{3-5}
+  render: function() {
+    return (
+      <ReactCSSTransitionGroup transitionName="example" transitionAppear="true">
+        <h1>Fading at Initial Mount</h1>
+      </ReactCSSTransitionGroup>
+    );
+  }
+```
+
+During the initial mount `ReactCSSTransitionGroup` will get the `example-appear` CSS class and the `example-appear-active` CSS class added in the next tick.
+
+```css
+.example-appear {
+  opacity: 0.01;
+  transition: opacity .5s ease-in;
+}
+
+.example-appear.example-appear-active {
+  opacity: 1;
+}
+```
+
+At the initial mount all children of the `ReactCSSTransitionGroup` will `appear` but not `enter`. However all children later added to an existing `ReactCSSTransitionGroup` will `enter` but not `appear`.
+
+> Note:
+>
+> The prop `transitionAppear` has been added to `ReactCSSTransitionGroup` in version `0.13`. In favor of reverse-compatibility the default value is set to `false`.
```

Comment:
"was added" (less passive voice :) ).

"To maintain backwards compatability, the default..."